source/common/runtime/runtime_features.cc

```diff
@@ -0,0 +1,37 @@
+#include "common/runtime/runtime_features.h"
+
+namespace Envoy {
+namespace Runtime {
+
+// Add additional features here to enable the new code paths by default.
```

Comment:
Sorry to harp on this, but I think what would help me/readers here is a short description of the behavior of what an entry in runtime_features and disallowed_features do. I know this is described elsewhere, but it's hard enough to keep track of (and important enough) that IMO we should replicate a short version here?